rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParserOnlyContribution.java

```diff
@@ -150,13 +151,13 @@ public CompletableFuture<IList> parseCodeActions(String commands) {
     }
 
     @Override
-    public InterruptibleFuture<IList> inlayHint(@Nullable ITree input) {
+    public InterruptibleFuture<IList> inlayHint(ITree input) {
         return InterruptibleFuture.completedFuture(VF.list());
     }
 
     @Override
     public InterruptibleFuture<ISourceLocation> prepareRename(IList focus) {
-        return InterruptibleFuture.completedFuture(null);
+        return InterruptibleFuture.completedFuture(URIUtil.unknownLocation());
```

Comment:
In `ParamtericTextDocumentService` there is still an explicit check for null instead of a check for an unknown location.